source/Reference.js

```diff
@@ -2,4 +2,14 @@ module.exports.GAME_WIDTH = 230
 module.exports.GAME_HEIGHT = 130
 module.exports.JOIN_GAME_TEXT = "PRESS TO JOIN"
 module.exports.JUNK_FREQUENCY_RANGE = { lower: 5, upper: 10}
-module.exports.TIME_TO_DESPAWN = 5 * 1000
\ No newline at end of file
+module.exports.TIME_TO_DESPAWN = 5 * 1000
+
+module.exports.ControlScheme = {
+    WASD: 0,
+    NUM_PAD: 1,
+    keys: {
+        0: {up: "W", down: "S", left: "A", right: "D", fire: "<space>", inUse: false},
+        1: {up: "<num-8>", down: "<num-2>", left: "<num-4>", right: "<num-6>", fire: "<num-0>", inUse: false}
+    }
+}
+module.exports.controlTypeCount = 2
```

Comment:
It's not indexing by player number, per se. The numerical designation is just to make it iteratable. Couldn't find another way to do that. Docs on enums in JavaScript are... lacking. As for the indexing by strings, I like keeping it there for readability and meaning. Giving a meaningful name to each control scheme, so to speak.

Grabbed length via the `Object` function though. Just subtracted 1 to get the correct size. I was trying to do this earlier, using a function that's supposed to be part of Enum, but I couldn't get that working. Didn't think to look more generically. Thanks!